operations/restrictions.md

```diff
@@ -369,3 +377,126 @@ Removes restrictions from the service.
 ```javascript
 {}
 ```
+
+## Set restrictions
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change.
+
+Adds new restrictions with the specified conditions.
+
+If there already exists a restriction with the same conditions, following scenarios apply:
+
```

Comment:
I think I would make the merging discussion a small separate section, by inserting a heading:

### Merging algorithms